### PR TITLE
Adds more box crafting types to cardboard 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Clearly the best idea

## About The Pull Request

Simply makes more types of boxes craftable via cardboard

## Why It's Good For The Game

I thought I already added these, I don't know why I didn't!
Anywho Its to help inventory management. As well as helps other people maybe know what your keeping at a glace in your bag! This also helps warden/barkeep get more thematic ammo boxes rather then using normal boxes/paper bags.

## Changelog
:cl:
add: cardboard box options
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Wait I'm on TG github!
This is why all the other ones I added are missing...

**Comment by optimumtact** [comment]
basedpoly

**Comment by Trilbyspaceclone** [comment]
image
Im not seeing the problem

**Comment by FantasticFwoosh** [comment]
Actual comment about this, I added the current set of unique limited boxes to crafting (egg, pizza, candle etc) but I stand opposed to this on the basis that half of this is bloat of item subtypes.

These only add a few pixel wide image across the front and bear no difference whatsoever. Couldn't you make the contents of the generic box reflect its appearance instead?

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -317,17 +317,29 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
  * Cardboard
  */
 GLOBAL_LIST_INIT(cardboard_recipes, list ( \
+	new/datum/stack_recipe("folder", /obj/item/folder), \
 	new/datum/stack_recipe("box", /obj/item/storage/box), \
 	new/datum/stack_recipe("light tubes", /obj/item/storage/box/lights/tubes), \
 	new/datum/stack_recipe("light bulbs", /obj/item/storage/box/lights/bulbs), \
+	new/datum/stack_recipe("disk box", /obj/item/storage/box/disks), \
+	new/datum/stack_recipe("pillbittle box", /obj/item/storage/box/pillbottles), \
```
Comment (nemvar) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: Spelling

**Comment by nemvar** [comment]
We also have the ability to create "folders" for crafting recipes. Maybe that would be a better option for all the fancy box types.

**Comment by FantasticFwoosh** [comment]
> We also have the ability to create "folders" for crafting recipes.

b-b-but folders are a singular object used for holding paper ingame, i dont know what you're talking about. 

What you could do is a multi choice optional screen when you edit a box with a pen like how you can currently already edit the box's name.

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: I wish I made this at not sleep-o-clock

**Comment by Trilbyspaceclone** [comment]
@FantasticFwoosh 
Ammo boxes limit the type of object that fits in them.
The other sub type boxes have different names and icons that can be used to organize things like chemistry, medical and such.

**Comment by Trilbyspaceclone** [comment]
44744 has this way better

**Pull request closed without merging**